plugins/wporg-5ftf/views/form-my-pledges.php

```diff
@@ -0,0 +1,92 @@
+<?php
```

Comment:
Could we call this `list-my-pledges.php` instead? Personally I wouldn't think of this as a "form", so this isn't where I would look for this code.

Reply:
That makes sense 👍